lib/brook/serializer.ex

```diff
@@ -13,7 +13,7 @@ defprotocol Brook.Serializer.Protocol do
   @doc """
   Convert the supplied Elixir term to an encoded term wrapped in an `:ok` tuple.
```

Comment:
docs should _probably_ reflect this change. if only you had an admin to do that type of grunt work for you.